walkthroughs/programming-syntax.md

```diff
@@ -72,7 +72,7 @@ The branch name can be anything you like, for example:
 Please enter a name for your Instant Answer:
 ```
 
-Type `regex1` (since *regex* already exists in the repository, we'll add a character for this tutorial). The tool will do the rest:
+Type `regex1` (since *regex* already exists in the repository, we'll add a character for this tutorial). If the name contains more than one words, separate them with a space. The tool will do the rest:
```

Comment:
I edited the file from github itself. Now I have fixed the grammar and committed the changes. The merge request is also updated.